ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml

```diff
@@ -15,6 +15,10 @@ base_images:
     name: "4.7"
     namespace: ocp
     tag: assisted-installer-controller
+  assisted-service:
+    name: "4.7"
+    namespace: ""
```

Comment:
ocp

Reply:
This is the output of using `make ci-operator-config` and the jobs will fail with the ocp namespace here.

Reply:
The promotion is defined under the assisted-service configuration for "ocp/4.7". No way it would work that way with an empty namespace.

Reply:
Sorry, I saw all the namespaces got bumped on https://github.com/openshift/release/pull/15597
You should change it accordingly to "4.8"

Reply:
Ok I made some changes on https://github.com/openshift/release/pull/15742 to avoid those numbers.
We should use `ocp-kni` namespace and `assisted` repository